server/waiter.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"encoding/base64"
+	"fmt"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes"
+	pb "github.com/googleapis/gapic-showcase/server/genproto"
+	lropb "google.golang.org/genproto/googleapis/longrunning"
+)
+
+var waiterSingleton Waiter = &waiterImpl{
+	nowF: time.Now,
+}
+
+func GetWaiterSingleton() Waiter {
```

Comment:
There's generally no need to do this. Just let others access the variable.

Reply:
Fixed!